misc/build.func

```diff
@@ -501,6 +502,41 @@ install_script() {
   fi
 }
 
+check_container_ressources() {
+  local required_ram="$var_ram"  # Expected RAM in MB
+  local required_cpu="$var_cpu"  # Expected number of Cores
+  local app_name="${APP}"        # Name of App
+
+  # Check actual RAM & Cores
+  local current_ram
+  current_ram=$(free -m | awk '/^Mem:/{print $2}')
+  current_cpu=$(nproc) 
+
+  # Check whether the current RAM is less than the required RAM or the CPU cores are less than required
+  if [[ "$current_ram" -lt "$required_ram" ]] || [[ "$current_cpu" -lt "$required_cpu" ]]; then
+    echo -e "\n⚠️${HOLD} ${GN}Required: ${required_cpu} CPU, ${required_ram}MB RAM ${CL}| ${RD}Current: ${current_cpu} CPU, ${current_ram}MB RAM${CL}"
+    echo -e "${YWB}Please ensure that the ${app_name} LXC is configured with at least ${var_cpu} vCPU and ${var_ram} MB RAM for the build process.${CL}\n"
+    exit 1  
+  else
+    echo -e ""
+  fi
+}
+
+check_storage() {
+  # Check if the /boot partition is more than 80% full
+  if (( $(df /boot | awk 'NR==2{gsub("%","",$5); print $5}') > 80 )); then
+    # Prompt the user for confirmation to continue
+    echo -e "⚠️${HOLD} ${YWB}Warning: Storage is dangerously low.${CL}"
+    echo -e "Continue anyway? <y/N>${CL}"
+    read -r -p "" prompt  
```

Comment:
```suggestion
    read -r -p "Continue anyway? <y/N>${CL}" prompt  
```
should this prompt be similar to the script prompts like this?

Reply:
I had it like this before, but the formatting was somehow strange.

Reply:
Perhaps it is the `${CL}` part that affected formatting, since that is not used in the script prompts?

Reply:
I can test tomorrow 2 or 3 variants. The best (ui/ux) wins :smile: